docs/examples/pool/template_DUAL.py

```diff
@@ -1,3 +1,27 @@
+# %%
+# .. note::
+#    The generated animation can be found at the bottom of the page.
+#
+# | **Google Colab Note**: If the notebook fails to run after installing the
+#   needed packages, try to restart the runtime (Ctrl + M) under
+#   Runtime -> Restart session.
+#
+# .. image:: https://colab.research.google.com/assets/colab-badge.svg
+#    :target: "$colab_link"
+#
+# | **Notebook Dependencies**
+# | Uncomment the following cells to install all dependencies for this
```

Comment:
Are there multiple cell*s* with dependencies? If not update this comment also for all other templates.